CHANGELOG.md

```
@@ -1 +1,3 @@
 ### v7.0.0
+
+### v7.0.0
```

Markdownlint (0.37.0)

Multiple headings with the same content

(MD024, no-duplicate-heading)